Failed tests (11286): TestPGPrepared

The following test appears to have failed:

[#11286](https://circleci.com/gh/cockroachdb/cockroach/11286):

```
E0113 22:02:44.386602 880 sql/pgwire/server.go:110  EOF
W0113 22:02:44.402635 880 sql/pgwire/v3.go:148  unrecognized configuration parameter "client_encoding"
W0113 22:02:44.402660 880 sql/pgwire/v3.go:148  unrecognized configuration parameter "extra_float_digits"
W0113 22:02:44.402673 880 sql/pgwire/v3.go:148  unrecognized configuration parameter "datestyle"
I0113 22:02:44.403864 880 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestPGPrepared (0.42s)
	pgwire_test.go:263: SELECT 3 + $1, $1 + $2: []interface {}{0, "a"}: expected error: pq: param $2 ("a"): unknown int value, got pq: param $2: strconv.ParseInt: parsing "a": invalid syntax
=== RUN   TestPrivilege
--- PASS: TestPrivilege (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
=== RUN   TestPrivilegeValidate
--- PASS: TestPrivilegeValidate (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
=== RUN   TestSystemPrivilegeValidate
--- PASS: TestSystemPrivilegeValidate (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
--
--- PASS: TestColumnTypeSQLString (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
=== RUN   TestInitialKeys
--- PASS: TestInitialKeys (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
FAIL
FAIL	github.com/cockroachdb/cockroach/sql	23.303s
=== RUN   TestNoLinkForbidden
--- PASS: TestNoLinkForbidden (0.11s)
=== RUN   TestDates
I0113 22:02:21.331881 873 rpc/clock_offset.go:153  monitoring cluster offset
I0113 22:02:21.333846 873 raft/raft.go:446  [group 1] 1 became follower at term 5
I0113 22:02:21.333938 873 raft/raft.go:234  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0113 22:02:21.333989 873 raft/raft.go:526  [group 1] 1 is starting a new election at term 5
I0113 22:02:21.334034 873 raft/raft.go:459  [group 1] 1 became candidate at term 6
I0113 22:02:21.334077 873 raft/raft.go:508  [group 1] 1 received vote from 1 at term 6
I0113 22:02:21.334126 873 raft/raft.go:487  [group 1] 1 became leader at term 6

```
Please assign, take a look and update the issue accordingly.

--- a/sql/pgwire_test.go
+++ b/sql/pgwire_test.go
@@ -228,7 +228,7 @@ func TestPGPrepared(t *testing.T) {
 		"SELECT 3 + $1, $1 + $2": {
 			base.Params("1", "2").Results(4, 3),
 			base.Params(3, "4").Results(6, 7),
-			base.Params(0, "a").Error(`pq: param $2 ("a"): unknown int value`),
+			base.Params(0, "a").Error(`pq: param $2: strconv.ParseInt: parsing "a": invalid syntax`),
 		},
 		// TODO(mjibson): test date/time types
 	}

